src/sage/combinat/tiling.py

```diff
@@ -1377,8 +1393,8 @@ def show3d(self, size=1):
 
         INPUT:
 
-        - ``self`` - a polyomino of dimension 3
-        - ``size`` - number (optional, default: ``1``), the size of each
+        - ``self`` -- a polyomino of dimension 3
+        - ``size`` -- number (optional, default: ``1``), the size of each
```

Comment:
we could remove the term `optional`. We have it many times for parameters with default value.

Reply:
I've left the "optional" as is, too many changes in this PR already